reduct-nepal/seep
# implemented feature of uploading and fetching audio from cloud rathe…

**Task Completed:**

- [x] Implemented the feature of uploading and fetching(playing audio files) from to and from firebase cloud.
- [x] Added url field into the audioInfo collection to store the url of the audio files

**To test:**
#CASE1
While giving Test
- Log into Reduct Nepal Testing Portal
- Give a practice test or actual test
- User should be able to give the test without any issue
#CASE2
While Uploading Audio
- Log into Reduct Nepal Testing Portal as Admin
- Try Uploading a audio files
- The audio files get uploaded to the Reduct Staging database . 
- Only after the audio gets uploaded the input field for entering master transcript get appeared.


**Expectations:**

- Normal workflow with no change of behavior from user point of view 

**Intentions:**

- To implement firebase cloud for storing audio files.


PS*-The audio files get uploaded to Reduct Staging database(used for testing purpose) while testing from local device,so please feel free to upload random files to test the functionality
